config/routes.rb

```diff
@@ -36,32 +40,47 @@
 
   # Simple Smart Answer pages
   get ":slug/y(/*responses)" => "simple_smart_answers#flow", :as => :smart_answer_flow
-  get ":slug", to: "simple_smart_answers#show", constraints: FormatRoutingConstraint.new('simple_smart_answer')
+  constraints FormatRoutingConstraint.new('simple_smart_answer') do
+    get ":slug", to: "simple_smart_answers#show"
+    get ":slug/:part", to: redirect('/%{slug}') # Support for simple smart answers that were once a format with parts
+  end
 
-  # Help pages
-  get "/help", to: "help#index"
-  get "/tour", to: "help#tour"
-  get "*slug", slug: %r{help/.+}, to: "help#show", constraints: FormatRoutingConstraint.new('help_page')
+  # Campaign pages.
+  get "/ukwelcomes", to: "campaign#uk_welcomes" # This is a hardcoded page in Frontend and does not have an associated artefact
+  constraints FormatRoutingConstraint.new('campaign') do
+    get ":slug", to: "campaign#show"
+    get ":slug/:part", to: redirect('/%{slug}') # Support for campaigns that were once a format with parts
```

Comment:
Weirdly [Transactions and Campaigns register exact routes](https://github.com/alphagov/publisher/blob/c1506fc0abeab2b5d88eb06582ab897e552b1db2/lib/registerable_edition.rb#L30) (along with Help pages) so a request for `:slug/:part` won't get as far as Frontend for pages with these formats.  I'm not sure why these ones in particular.